index.html

```diff
@@ -10,25 +10,9 @@
 			body {background:black;color:white;height:100%;width:100%;margin:0;}
 			input, textarea {-webkit-user-select:text;}
 		</style>
-		<!--
-		<script src="lib/tweenjs-0.6.0.min.js"></script>
-		<script src="lib/easeljs-0.8.0.min.js"></script>
-		<script src="lib/preloadjs-0.6.0.min.js"></script>
-		<script src="lib/jquery-2.1.0.min.js"></script>
-		<script src="lib/grid-0.1.js"></script>
-		<script src="lib/Main.js"></script>
-		<script src="lib/EyeCandy.js"></script>
-		<script src="lib/LoaderBar.js"></script>
-		<script src="lib/MapContainer.js"></script>
-		<script src="lib/PanelContainer-0.3.js"></script>
-		<script src="lib/Slider.js"></script>
-	-->
 		<script src="https://code.createjs.com/createjs-2015.11.26.min.js"></script>
 		<script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
-		<script src="lib/bitmapdata-1.1.1.min.js"></script>
-		<script src="lib/canvas-toBlob.js"></script>
-		<script src="lib/GameObjects-0.3.js"></script>
-		<script src="lib/TI3SA-0.1.min.js"></script>
+		<script src="TI3SA.min.js"></script>
```

Comment:
You removed the loading of these scripts, and it still works?

Reply:
Everything is packed together when minified. So those would no longer be required.

Reply:
I just saw that there were still two files that where included but also in the minified version. so I removed the CDN files